### PR TITLE
GameDB: Several game fixes and a gamelist entry correction

### PR DESCRIPTION
**Pull request by SeraphisCain**
### Description of Changes
1. Several game fixes:

Space Sheriff Spirits (Fixes black stage backgrounds):

Before:
gs_20220415071850_Space Sheriff Spirits _NTSC-J__SLPS-25649

After:
gs_20220415071905_Space Sheriff Spirits _NTSC-J__SLPS-25649

Simple 2000 Series Vol. 114 - The Onna Okappichi Torimonochou - Oharu-chan GoGoGo (Fixes black screen in-game.)

Before:
gs_20220415072219_Simple 2000 Series Vol  114 - The Onna Okappichi Torimonochou - Oharu-chan GoGoGo_ _NTSC-J__SLPS-20489

After:
gs_20220415072233_Simple 2000 Series Vol  114 - The Onna Okappichi Torimonochou - Oharu-chan GoGoGo_ _NTSC-J__SLPS-20489

Demon Chaos (Fixes black screen on boot, also note that the same issue likely exists in the NTSC-J release, Ikusa Gami, but I don't currently have that available to test).

Before:
gs_20220415072531_Demon Chaos _PAL-M5__SLES-54305

After:
gs_20220415072830_Demon Chaos _PAL-M5__SLES-54305

C1 Grand Prix: Racing Battle (Fixes black lines when upscaling):

Before:
gs_20220415073046_C1 Grand Prix Racing Battle _NTSC-J__SLPM-65897

After:
gs_20220415073104_C1 Grand Prix Racing Battle _NTSC-J__SLPM-65897

Keroro Gunso - Mero Mero Battle Royale (Fixes invisible text):

Before:
gs_20220415075050_Keroro Gunsou - Mero Mero Battle Royale _NTSC-J__SLPS-25399

After:
gs_20220415075106_Keroro Gunsou - Mero Mero Battle Royale _NTSC-J__SLPS-25399

2. Fixed an erroneous game entry:

So this is incidental, but when I was adding the fix for C1 Grand Prix, I noticed it was listed twice despite only having one release. It turns out the second listing had a very similar serial number to the actual release (SLPM-65879 instead of SLPM-65897), so I went ahead and filled in the correct game title for that serial number (a re-release of a Love Hina game).

### Rationale behind Changes
Solves several game issues.

### Suggested Testing Steps
Test the games.

Note: Sorry for the two commits here. I forgot a space in one of the comments, went back to fix it (and made the commit description a bit more clear), and after half an hour of Googling am still no closer to understanding how to squash and merge before submitting a PR. Apologies!

### CODE REVIEW

File: bin/resources/GameIndex.yaml [hunk 1]
```diff
@@ -31957,6 +31961,8 @@ SLPS-20488:
 SLPS-20489:
   name: "Simple 2000 Series Vol. 114 - The Onna Okappichi Torimonochou - Oharu-chan GoGoGo!"
   region: "NTSC-J"
+  gsHWFixes:
+    preloadFrameData: 1 # Fixes black screen in-game
```
Comment (RedDevilus) [inline bin/resources/GameIndex.yaml:31965]: Missing punctation in the comment.

Comment (SeraphisCain) [reply bin/resources/GameIndex.yaml:31965]: Can't believe I missed that. Fixed.

**Review by Mrlinkwii — APPROVED** [review]

File: bin/resources/GameIndex.yaml [hunk 2]
```diff
@@ -33333,6 +33339,8 @@ SLPS-25398:
 SLPS-25399:
   name: "Keroro Gunsou - Mero Mero Battle Royale"
   region: "NTSC-J"
+  gsHWFixes:
+    preloadFrameData: 1 # Fixes invisible text.
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:33343]: trailing space

File: bin/resources/GameIndex.yaml [hunk 3]
```diff
@@ -17336,6 +17336,8 @@ SLES-54305:
   name: "Demon Chaos"
   region: "PAL-M5"
   compat: 5
+  gameFixes:
+    - VIF1StallHack # Fixes black screen on boot.
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:17340]: trailing space

Comment (SeraphisCain) [reply bin/resources/GameIndex.yaml:17340]: Fixed.

Comment (refractionpcsx2) [reply bin/resources/GameIndex.yaml:17340]: And the other one?

**Pull request merged**